src/main/java/me/axieum/mcmod/authme/mixin/AbuseReportContextAccessor.java

```diff
@@ -0,0 +1,21 @@
+package me.axieum.mcmod.authme.mixin;
+
+import net.minecraft.client.report.AbuseReportContext;
+import net.minecraft.client.report.ReporterEnvironment;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+/**
+ * Provides the means to access protected members of the Splash Text Resource supplier.
```

Comment:
I think this is meant to refer to something else?

Reply:
yes, forgot to update/remove the details from the copied file - oopsie